configuration_popup_filter.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "AdGuard DNS filter for popup domains",
+  "description": "A filter comprising elements from AdGuard filters, EasyList, ABPindo and designed to enhance compatibility with popup blocking at the DNS level. It is crafted to provide the address of a blocking page in response to domain queries.",
+  "homepage": "https://github.com/AdguardTeam/AdguardSDNSFilter",
+  "license": "https://github.com/AdguardTeam/AdguardSDNSFilter/blob/master/LICENSE",
+  "sources": [
+    {
+      "name": "AdGuard Base filter ad servers",
+      "source": "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/BaseFilter/sections/adservers.txt",
+      "type": "adblock",
+      "exclusions_sources": ["Filters/exclusions.txt"],

```

Comment:
HostlistCompiler allows having common `inclusions` and `exclusions` on the same level as `sources`, there's no need to duplicate them here.